Gem/Code/Source/SampleComponentManager.cpp

```diff
@@ -1546,7 +1546,7 @@ namespace AtomSampleViewer
         // Create and register the rhi scene with only feature processors required for AtomShimRenderer (only for AtomSampleViewerLauncher)
         RPI::SceneDescriptor sceneDesc;
         sceneDesc.m_nameId = AZ::Name("RHI");
-        sceneDesc.m_featureProcessorNames.push_back("AuxGeomFeatureProcessor");
```

Comment:
This has already been fixed. https://github.com/o3de/o3de-atom-sampleviewer/pull/490